### ISSUE TITLE
Settings - Change name and email

### ISSUE THREAD

**Issue by Loganrfs** [DevLAN-io/RFSPager-issues#15]
In  settings have a setting to change your name and email 

so people can edit there profile in the settings 

like there name / email and photo

**Comment by nicko170** [comment]
Yeah, and the ability to convert an anon account into a real account too.
